src/components/AddressSearch.js

```diff
@@ -117,7 +117,18 @@ const AddressSearch = (props) => {
             sublocality: 'long_name',
             postal_code: 'long_name',
             administrative_area_level_1: 'short_name',
-            country: 'long_name',
+            country: 'short_name',
+        });
+
+        /**
+         * The short_name needs to be utilised in the StatePicker but we also
+         * need the full name of the state when the country selected is not USA, that's why
+         * this is called twice.
+         */
```

Comment:
We should use `//` for comments inside functions, `/**` for comments about functions.

I also suggested a slight rewording that I think is more clear, let me know what you think:

```suggestion
        // The state's iso code (short_name) is needed for the StatePicker component but we also
        // need the state's full name (long_name) when we render the state in a TextInput.
```